extensions/php/languages/phpdoc/highlights.scm

```diff
@@ -0,0 +1,13 @@
+(tag_name) @keyword
+
+(tag (variable_name) @variable)
+(variable_name "$" @operator)
+
+(tag
+  (tag_name) @keyword
+  (#eq? @keyword "@method")
+  (name) @function.method)
+
+(primitive_type) @type.builtin
+(named_type (name) @type) @type
+(named_type (qualified_name) @type) @type
```

Comment:
Most of these were copied from the php highligts: https://github.com/zed-industries/zed/blob/main/extensions/php/languages/php/highlights.scm#L6-L9

The `@method` highlight was taken from neovim, and allows the highlighting of `@method bool insufficient()` to match that of `function insufficient()`.

Reply:
BTW the neovim highlight queries are at https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/phpdoc/highlights.scm. I don't know or use neovim, but I found that most of them were more specific than was needed just to get highlighting working (except for `@method` as mentioned above)